types/generated/components.d.ts

```diff
@@ -806,6 +821,22 @@ export interface SharedTabCardTheme extends Schema.Component {
   };
 }
 
+export interface SharedTimelineCard extends Schema.Component {
+  collectionName: 'components_shared_timeline_cards';
+  info: {
+    displayName: 'TimelineCard';
+  };
+  attributes: {
+    Title: Attribute.String &
+      Attribute.SetMinMaxLength<{
+        maxLength: 256;
+      }>;
+    Text: Attribute.Text & Attribute.Required;
+    Image: Attribute.Media<'images' | 'files' | 'videos' | 'audios', true> &
```

Comment:
Question:
Why do we have `'images' | 'files' | 'videos' | 'audios', true` instead of just images and, probably videos?

Reply:
I've used basic image component from stapi, it's included all options which you mentioned above